website/static/_redirects

```diff
@@ -742,6 +743,9 @@ https://tutorial.getdbt.com/* https://docs.getdbt.com/:splat 301!
 /v0.13/docs/requiring-dbt-versions / 301
 /v0.14/docs/cloud-developer-ide / 301
 
+/v0.13/docs/building-a-new-adapter /guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter 301
```

Comment:
This redirect is 404ing. However, I'm confused as to why, because the correct destination *should be* `/guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter`. That page resolves correctly for me.

If you enter this URL from the chain mentioned in the spreadsheet (row 989), it also resolves... to `https://deploy-preview-3284--docs-getdbt-com.netlify.app/guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter`!

Searching the raw file, it seems like there are multiple different redirects that involve this page. I noticed this one is lacking a space between the destination URL and the redirect code, and that might also cause a problem:

`/docs/building-a-new-adapter /guides/advanced/adapter-development/3-building-a-new-adapter301`

Might also want to make sure the redirects are being processed in the correct order, too. Keep in mind that the file is processed from top to bottom, and if Netlify finds a matching rule, it immediately navigates away without processing the rest of the rules. So we want to make sure the redirects go in order from specific -> general.

@runleonarun can you confirm what the canonical URL/final destination should be? I'm assuming it's `/guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter`.

If you're having trouble figuring out what's going on here, may I refer you to @JKarlavige for help?